arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/imc/mmu/opcode/CodeCopy.java

```diff
@@ -38,7 +38,8 @@ public CodeCopy(final Hub hub) {
     this.hub = hub;
     this.contract = hub.currentFrame().metadata();
 
-    this.targetId(hub.currentFrame().contextNumber())
+    this.exoBytes(Optional.of(hub.romLex().getCodeByMetadata(contract)))
```

Comment:
Here `exoBytes` is made to contain (nothing or) the currently executing bytecode, correct ?

Reply:
Correct